@types/globals.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'lodash' {
```

Comment:
This definitely works for small dependency usage. Optionally you can add packages like: https://www.npmjs.com/package/@types/lodash to do all the heavy lifting, but it will add to your dependency list. Typescript's module resolution will find anything placed in `node_modules/@types`, which is where the dependency will be stored.

Reply:
Yeah, I didn't want to pull more dependencies, at least not for the lodash, as it's usage is minimal in the codebase and we are in the process of getting rid of it. Modern ES allows to achieve most of that natively.